### ISSUE TITLE
Add additional links to admin menu

### ISSUE THREAD

**Issue by jywarren** [publiclab/plots2#1167]
Hi, this is a [first-timers-only issue](https://github.com/publiclab/plots2/projects/2). This means we've worked to make it more legible to folks who either **haven't contributed to our codebase before, or even folks who haven't contributed to open source before**. 

If that's you, we're interested in helping you take the first step and can answer questions and help you out as you do. Note that we're especially interested in contributions from people from groups underrepresented in free and open source software!

If you have contributed before, consider leaving this one for someone new, and looking through our general [help wanted](https://github.com/publiclab/plots2/labels/help-wanted) issues. Thanks!

## The problem
Our "admin" menu, shown to "admin" users, could use a couple more items:

* link to https://publiclab.org/wiki/locked "Locked pages"
* link to spam moderation https://publiclab.org/spam "Spam moderation"
* link to users list https://publiclab.org/people "Contributors"

Here's what it looks like now -- see how "Email search" appears just below a double grey line:

![screen shot 2017-01-04 at 1 04 54 pm](https://cloud.githubusercontent.com/assets/24359/21653305/6ab4e2ba-d27e-11e6-9899-16ecde430811.png)

https://github.com/publiclab/plots2/blob/master/app/views/layouts/_header.html.erb#L178-L179

```ruby
<li class="divider"></li>
<li><a href="/useremail">Email search</a></li> 
```

## Solution

First, the two lines need to be swapped, since the divider is currently above, rather than below, the existing item.

Second, each of the items listed above should be added as a new line in a new entry similar to `<li><a href="/useremail">Email search</a></li>`

just below the existing "Email search" line. 

## Steps to Fix

- [x] claim this issue with a comment here, below, and ask any clarifying questions you need
- [ ] set up a repository locally following the README instructions, and make sure that all tests pass
- [ ] try to fix the issue following the steps above, but even before you're done, you can:
- [ ] commit your changes and start a pull request (see [contributing to Public Lab software](https://publiclab.org/wiki/contributing-to-public-lab-software)) but mark it as "in progress" if you have questions or if you haven't finished
- [ ] alert someone via the developers list (see below) to have your pull request merged. This may take a few extra steps depending on when you do it, but we'll help you out!

Please email the developers list (see https://publiclab.org/wiki/developers) if you have questions, and take a look at [our first-timers landing page for more information](https://github.com/publiclab/plots2/projects/2)!

**Comment by krys2fa** [comment]
Hello Jywarren, I would like to work on this issue. Kindly let me know how to proceed further.

**Comment by ananyo2012** [comment]
Oh this was very fast! :smile: First-timer issues are really drying up fast! @krys2fa Install the app according to the steps given in readme first. Then add the links in header.

**Comment by jywarren** [comment]
Very happy to have your contribution, please ask if you need help!

**Comment by krys2fa** [comment]
Thanks jywarren, i've just submitted a pull request. Let's see how it goes. :)
I've been waiting for a first-timer issue to contribute to for some time now. Lol

**Comment by ananyo2012** [comment]
@krys2fa Great! :smile: Left a review!

**Comment by kanpeki** [comment]
Hi, so is this issue unavailable now? I see that no one is assigned yet, even though someone offered to help out. I would like to contribute, but it's my first time and I am unsure of the process (aka if it's OK to work on this issue or not). Please let me know.

**Comment by ananyo2012** [comment]
@kanpeki This is already claimed. Actually you can't assign someone a issue who isn't a member of the org. That's why we use a first-timer issue template which has a checklist. You can have a look at other  [fto-candidate](https://github.com/publiclab/plots2/labels/fto-candidate) issues. They are close to being first-timer issues.

**Comment by kanpeki** [comment]
@ananyo2012 Ok, will do. Thank you for the prompt answer.

**Comment by hitesharora19** [comment]
Hello, I would like to work in this issue.

**Comment by ananyo2012** [comment]
This issue is claimed. You can have a look at other [fto-candidate](https://github.com/publiclab/plots2/labels/fto-candidate) issues. They are close to being first-timer issues.

**Comment by jywarren** [comment]
Hello, this is already being resolved in #1167. Thank you!
